go.sum

```diff
@@ -23,6 +23,7 @@ github.com/google/uuid v0.0.0-20161128191214-064e2069ce9c h1:jWtZjFEUE/Bz0IeIhqC
 github.com/google/uuid v0.0.0-20161128191214-064e2069ce9c/go.mod h1:TIyPZe4MgqvfeYDBFedMoGGpEw/LqOeaOT+nhxU+yHo=
 github.com/gruntwork-io/gruntwork-cli v0.1.2 h1:lUaJdPTRJ0HGELW8a3r2QJnHA+Dak+PkvOP+T8ykXF4=
 github.com/gruntwork-io/gruntwork-cli v0.1.2/go.mod h1:IBX21bESC1/LGoV7jhXKUnTQTZgQ6dYRsoj/VqxUSZQ=
+github.com/gruntwork-io/gruntwork-cli v0.7.0 h1:YgSAmfCj9c61H+zuvHwKfYUwlMhu5arnQQLM4RH+CYs=
```

Comment:
Are these version number bumps intentional?

Reply:
They happened automatically - so I've left them in.